### DISCUSSION TITLE
Convert Migration from Laravel to Lucid

### DISCUSSION THREAD

**Discussion by tuliovargas** [adonisjs/core#2753] [Help]
Hi Experts,
is there a lib or way somehow to auto-convert migration from Laravel to Lucid?

**Comment by MadhuSaini22** [comment]
Yes, u can convert a Laravel application architecture to a Lucid micro-service. 
I can give you the over all  explanation and u can refer this provided below link which will clear your concept regarding your [project!!]
(https://tech.vinelab.com/convert-laravel-app-architecture-to-lucid-a260541c411b)
Hope this will help you!!!
Happy coding :)
